go.mod

```diff
@@ -59,3 +59,5 @@ require (
 )
 
 replace vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
+
+replace github.com/openshift/api => github.com/gabemontero/api v0.0.0-20210910154316-eaa29e1f16df
```

Comment:
this captures the latest state of https://github.com/openshift/api/pull/1007

when that merges, we'll just change the version for openshift/api up above